defaults/main.yml

```diff
@@ -5,18 +5,29 @@ homebrew_prefix: /usr/local
 homebrew_install_path: "{{ homebrew_prefix }}/Homebrew"
 homebrew_brew_bin_path: /usr/local/bin
 
-homebrew_installed_packages:
-  - ssh-copy-id
-  - pv
+# List of packages to be installed, e.g.
+#   homebrew_installed_packages:
+#     - ssh-copy-id
+#     - pv
+homebrew_installed_packages: []
 
 homebrew_upgrade_all_packages: no
 
-homebrew_taps:
```

Comment:
Ah-- I missed the comment. That's fine then. While I do like having `homebrew_taps` explicitly matching what gets tapped, it is probably fine to leave it empty as long as it is well documented.